scalardl-web-client-sdk.js

```diff
@@ -32,9 +32,11 @@ class ClientService {
     /** @const */
     this.properties = properties;
     /** @const */
-    this.serverHost = properties['scalar.ledger.client.server_host'];
+    this.serverHost = this.getRequiredProperty_(properties,
```

Comment:
Is this `_` a style for internal method or something ?

Reply:
Exactly, private methods should end with a '_' character. 
cf :  https://google.github.io/styleguide/jsguide.html#naming-method-names